planning/obstacle_stop_planner/src/node.cpp

```diff
@@ -1518,7 +1533,10 @@ void ObstacleStopPlannerNode::resetExternalVelocityLimit(const double current_ac
 void ObstacleStopPlannerNode::publishDebugData(
   const PlannerData & planner_data, const double current_acc)
 {
-  const auto & current_vel = current_velocity_ptr_->twist.twist.linear.x;
+  mutex_.lock();
+  const auto current_vel = current_velocity_ptr_->twist.twist.linear.x;
+  mutex_.unlock();
+
```

Comment:
I think it is better to add current_vel as a argument
```suggestion
void ObstacleStopPlannerNode::publishDebugData(
  const PlannerData & planner_data, const double current_acc, const double current_vel)
  {
```

Reply:
Fixed in https://github.com/autowarefoundation/autoware.universe/pull/1330/commits/a472dd9417fbf6fc1b17998501f87a44e393ebdb